### PR TITLE
changes to action menu

### PR DESCRIPTION
**Pull request by jankrzempek**
Removed for now unused button from session menu

### CODE REVIEW

**Review by smialko — CHANGES_REQUESTED** [review]
Fixed session shouldn'y have "finish recording" as it can't be finished. We can remove this action button from fixed session for now :)

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 1]
```diff
@@ -30,6 +30,7 @@ struct SessionHeaderView: View {
                 Spacer()
                 if session.type == .fixed {
                     actionsMenuFixed
+                        .disabled(true)
```
Comment (smialko) [inline AirCasting/SessionViews/SessionHeaderView.swift:33]: maybe it would be better to remove this actionsMenuFixed at all for the time being?

**Review by smialko — CHANGES_REQUESTED** [review]
🙈

File: AirCasting/SessionViews/SessionHeaderView.swift (hunk 1 above)
Comment (jankrzempek) [reply AirCasting/SessionViews/SessionHeaderView.swift:33]: I really like this idea - done

File: AirCasting/CoreData/SessionEntity.swift [hunk 2]
```diff
@@ -132,3 +132,9 @@ extension SessionEntity {
     @NSManaged public func removeFromMeasurementStreams(_ values: NSSet)
 
 }
+
+extension SessionEntity {
```
Comment (smialko) [inline AirCasting/CoreData/SessionEntity.swift:136]: wait, we already have such extension with different session types

Comment (smialko) [reply AirCasting/CoreData/SessionEntity.swift:136]: Screenshot 2021-08-20 at 14 48 10

Comment (smialko) [reply AirCasting/CoreData/SessionEntity.swift:136]: could you move isActive there?

Comment (jankrzempek) [reply AirCasting/CoreData/SessionEntity.swift:136]: for sure, it is a bit mixed 😅

**Review by smialko — APPROVED** [review]

**Pull request merged**